ThunderCloud/StormLanguageController.swift

```diff
@@ -16,13 +16,14 @@ import UIKit
 /// `Locale` comes in the format of "en_GB" (Language, Region)
 /// `Locale` is able to ingest locales in the three letter format provided they are in the language_region format.
 /// This controller often re-organises the Storm file names to be in the language_region format before converting to `Locale`, once these are converted to `Locale` they can easily be compared with `Locale`s from the users device to find a match.
-public class StormLanguageController: NSObject {
```

Comment:
Whilst we're here, can we see if it's possible to remove the `NSObject` superclass?

Reply:
This is on a feature branch way back so it's unlikely it could be removed. Bit pointless as it's so far behind and we should do that change further upstream IMO

Reply:
Ahh fair, didn't realise that haha!